src/plugins/console/server/lib/spec_definitions/json/generated/nodes.info.json

```diff
@@ -27,6 +27,6 @@
         "transport"
       ]
     },
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/cluster-nodes-info.html"
+    "documentation": "https://opensearch.org/docs/latest/opensearch/index/#clusters-and-nodes"
```

Comment:
This is not the documentation for the Nodes info API (`/_nodes`). I don't think that documentation exists in the OpenSearch docs. We can leave this link, but add to `noDocumentation`